packages/manager/apps/pci-load-balancer/src/pages/detail/pools/detail/members/edit/Edit.page.tsx

```diff
@@ -0,0 +1,126 @@
+
+export default function EditPage() {
+  const { addSuccess, addError } = useNotifications();
+  const navigate = useNavigate();
+  const { t: tEdit } = useTranslation('pools/members/edit');
+  const { t: tPciCommon } = useTranslation('pci-common');
+  const { projectId, poolId, region, memberId } = useParams();
+  const onClose = () => {
+    navigate('..');
+  };
+
+  const { data: poolMember, isPending: isPendingPoolMember } = useGetPoolMember(
+    projectId,
+    poolId,
+    region,
+    memberId,
+  );
+  const [memberName, setMemberName] = useState('');
+  const [isTouched, setIsTouched] = useState(false);
+  const {
+    updatePoolMemberName,
+    isPending: isPendingUpdate,
+  } = useUpdatePoolMember({
+    projectId,
+    region,
+    poolId,
+    memberId,
+    name: memberName,
+    onError(error: ApiError) {
+      addError(
+        <Translation ns="octavia-load-balancer">
+          {(_t) =>
+            _t('octavia_load_balancer_global_error', {
+              message: error?.response?.data?.message || error?.message || null,
+              requestId: error?.config?.headers['X-OVH-MANAGER-REQUEST-ID'],
+            })
+          }
+        </Translation>,
+        true,
+      );
+      onClose();
+    },
+    onSuccess() {
+      addSuccess(
+        <Translation ns="pools/members">
+          {(_t) =>
+            _t('octavia_load_balancer_pools_detail_members_edit_name_success')
+          }
+        </Translation>,
+        true,
+      );
+      navigate('..');
+    },
+  });
+
+  const onConfirm = () => {
+    updatePoolMemberName();
+  };
+
+  const onCancel = () => {
+    navigate('..');
+  };
+  useEffect(() => {
+    if (poolMember) {
+      setMemberName(poolMember.name);
+    }
+  }, [poolMember]);
+  return (
+    <PciModal
+      title={tEdit('octavia_load_balancer_pools_members_edit_name_title')}
+      onClose={onClose}
+      onCancel={onCancel}
+      onConfirm={onConfirm}
+      type="default"
+      isPending={isPendingPoolMember || isPendingUpdate}
+      isDisabled={!memberName.trim()}
+      submitText={tEdit(
+        'octavia_load_balancer_pools_members_edit_name_confirm',
+      )}
+      cancelText={tEdit('octavia_load_balancer_pools_members_edit_name_cancel')}
+    >
+      <OsdsFormField
+        className="mt-8"
+        error={
+          isTouched && !memberName.trim()
+            ? tPciCommon('common_field_error_required')
+            : ''
+        }
+      >
+        <LabelComponent
+          text={tEdit('octavia_load_balancer_pools_members_edit_name_label')}
+          slot="label"
+          hasError={isTouched && !memberName.trim()}
+        />
+        <OsdsInput
+          value={memberName}
+          type={ODS_INPUT_TYPE.text}
+          className={
+            isTouched && !memberName.trim()
+              ? 'bg-red-100 border-red-500 text-red-500 focus:text-red-500'
+              : 'border-color-[var(--ods-color-default-200)] bg-white'
+          }
```

Comment:
<OsdsInput /> already have an `error` prop to handle error state
Also why do you change its regular state style ?